### PR TITLE
Re-enabling gaiaxpy

### PR DESCRIPTION
**Pull request by stvoutsin**
Change to re-enable gaiaxpy installation, and notes on tests with gaiaxpy enabled.

### CODE REVIEW

**Comment by Zarquan** [comment]
If we have a plan or issue for an automated test already in progress elsewhere, then this is OK to merge.
If not then can we add an automated test here ?

**Review by Zarquan — CHANGES_REQUESTED** [review]
If we have a plan/issue for an automatic test in progress elsewhere then this is good to go.
If we don't have an automated test yet, then can we add one here ?

**Comment by Zarquan** [comment]
If we already have an automated test for our Python libraries, then adding this one to the list should be enough.
If we have an active issue in progress to add an automated test for our Python libraries, then adding this one to the list should be enough.

**Review by Zarquan — COMMENTED** [review]
Is this a fair summary:

* Using dependency versions from GaiaXPy.
  210 passed, 3 warnings

* Using dependency versions from Aglais.
  1 failed, 209 passed, 121 warnings

* Using dependency versions from Aglais but  downgrading pandas to version from GaiaXPy.
  210 passed, 4160 warnings

Questions:
* What does this do to our own test notebooks ?
* How bad are the 4160 warnings ?

File: notes/stv/20220131-gaiaxpy-deploy-benchmark.txt [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+# -----------------------------------------------------
+# Test importing gaiaxpy
+
+%spark.pyspark
+import gaiaxpy
+
+> WARNING: IERSStaleWarning: leap-second file is expired. [astropy.utils.iers.iers]
+> FINISHED   
+> Took 2 sec. Last updated by admin at January 31 2022, 12:35:15 PM.
```
[the remark this answers is not in the document]
Comment (stvoutsin) [reply notes/stv/20220131-gaiaxpy-deploy-benchmark.txt:175]: I have a notebook that test the versions of the installed libraries, which we could include in the notebooks.json config file that is included in this repo. However we probably should first figure out which versions of the different packages will be installed, and whether we will included gaiaxpy or not due to the package compatibility issues.

File: notes/stv/20220131-gaiaxpy-deploy-benchmark.txt [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+# https://docs.google.com/spreadsheets/d/1w_acgDncwtqqBBXd34mRBytCcS8PYLaDKC-IwUzZ9fY/edit#gid=0
+
+
+# -----------------------------------------------------
+# Test importing gaiaxpy
+
+%spark.pyspark
+import gaiaxpy
+
+> WARNING: IERSStaleWarning: leap-second file is expired. [astropy.utils.iers.iers]
```
[the remark this answers is not in the document]
Comment (stvoutsin) [reply notes/stv/20220131-gaiaxpy-deploy-benchmark.txt:173]: I have only seen it when importing gaiaxpy, although the exception is triggered in an astropy module. Here's a github issue that seems related: https://github.com/astropy/astropy/issues/10228

[the remark this answers is not in the document]
Comment (Zarquan) [reply notes/stv/20220131-gaiaxpy-deploy-benchmark.txt:173]: Is it a problem, or can we safely ignore it ?

**Comment by stvoutsin** [comment]
> Is this a fair summary:
> 
> * Using dependency versions from GaiaXPy.
>   210 passed, 3 warnings
> * Using dependency versions from Aglais.
>   1 failed, 209 passed, 121 warnings
> * Using dependency versions from Aglais but  downgrading pandas to version from GaiaXPy.
>   210 passed, 4160 warnings
> 
> Questions:
> 
> * What does this do to our own test notebooks ?
> * How bad are the 4160 warnings ?

Yes that's correct. I can run a test with only pandas downgraded and see how our notebooks behave, but we'd still be downgrading a package which seems ugly

File: notes/stv/20220131-gaiaxpy-deploy-benchmark.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (Zarquan) [reply notes/stv/20220131-gaiaxpy-deploy-benchmark.txt:175]: So the library version test could include gaiaxpy in the list of things it checks ?

[the remark this answers is not in the document]
Comment (Zarquan) [reply notes/stv/20220131-gaiaxpy-deploy-benchmark.txt:175]: If so, then we can do that in a new issue/pr thread.

**Comment by Zarquan** [comment]
Yep, we need to test to see if the rest of our notebooks work with the downgraded pandas version. If they don't then that isn't really an option.

File: notes/stv/20220131-gaiaxpy-deploy-benchmark.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (stvoutsin) [reply notes/stv/20220131-gaiaxpy-deploy-benchmark.txt:173]: I've seen this again today when checking the output of some of our Public Examples notebooks (Mean Proper Motions), the error appeared in the following cell.
https://github.com/wfau/aglais-notebooks/blob/40f85ed8c7585f43b0431a1b1297bfb64df12961/AglaisPublicExamples/Mean%20proper%20motions%20over%20the%20sky_2GSNYBDWB.zpln#L78
So perhaps we need to look into it?

**Review by Zarquan — APPROVED** [review]

**Pull request merged**